lib/parse.js

```diff
@@ -1,87 +1,73 @@
 "use strict";
 
 var babylonToEspree = require("./babylon-to-espree");
-var parse = require("babylon").parse;
 var tt = require("babylon").tokTypes;
```

Comment:
Should we consider exporting the `babylon` module namespace from `babel-core`? The only reason not to is if we were going to version them independently, and realistically that doesn't seem likely.

Reply:
Then we could ditch the `babylon` dependency and not risk them being out of sync, right?